app/policies/provider_policy.rb

```diff
@@ -0,0 +1,12 @@
+class ProviderPolicy
```

Comment:
The other thing we should implement here is the [policy scope](https://github.com/varvet/pundit#scopes). Were you saving this for a later PR?